dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1730,7 +1729,31 @@ a.download-video {
   .devise_links {
     padding-left: 20px;
     margin-top: 20px;
-    border-left: 1px solid $black;
+  }
+}
+
+.devise_link_section {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  .or_break {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-right: 20px;
+    margin-left: -20px;
+    .vertical_bar {
+      width: 2px;
+      height: 100%;
+      background-color: $black;
```

Comment:
The screenshot looks like it's pure black, too, but we rarely use pure black for such a piece of UI.  Probably `$dark_charcoal` will be the colour to use.

Reply:
As discussed online, probably lighter grey will work better.  We can interate.